javers-persistence-sql/src/test/groovy/org/javers/repository/sql/domain/CommitRepositoryTest.groovy

```diff
@@ -0,0 +1,31 @@
+package org.javers.repository.sql.domain
+
+import org.javers.core.commit.CommitId
+import org.javers.core.commit.CommitMetadata
+import org.joda.time.LocalDateTime
+
+/**
+ * @author pawel szymczyk
+ */
+class CommitRepositoryTest extends BaseRepositoryTest {
+
+    def "should save and next find persisted commit metadata"() {
```

Comment:
masz racje, test pokazuje tylko ze przy save nowego obiektu generowany jest id i później podczas próby ponownego zapisu tego obiektu zwracany jest ten sam id. Na chwilę obecną tylko to jest nam potrzebne, nie wydaje mi sie żeby każde repo potrzebowało implementować metodę find